packages/components/src/components/navbar/Usage.mdx

```diff
@@ -0,0 +1,179 @@
+
+
+### Placement by priority
+The table below gives you an overview and guidance how to prioritize your secondary navigation items.
+
+| Breakpoint | Logo      | Application name | Main navigation | Secondary navigation | Search field |
+|------------|-----------|------------------|-----------------|----------------------|--------------|
+| ≥1440px    | Mandatory | High             | High            | High                 | High         |
+| ≥1200px    | Mandatory | High             | High            | High                 | Low          |
+| ≥1024px    | Mandatory | High             | n/c             | High                 | n/c          |
+| ≥720px     | Mandatory | Low              | n/c             | High                 | n/c          |
+| <720px     | Mandatory | Low              | n/c             | High                 | n/c          |
+
+## Breakpoints
+<img src={HeaderOverviewBreakpoints} alt="Header Breakpoints" />
+
+**≥1440px**\
+A large header with lots of space and the possibility to display each element.
+
+
+**≥1200px**\
+At this width you can decide by priority what to display. 
+
+
+**≥1024px**\
+At this width you should change your visible search bar into an icon with a label localized in the secondary navigation. Furthermore it is possible to move less important text links into a collapsible "More" text link.
+
+
+**≥720px**\
+At this width the header is simplified even further. Text links are moved into the burger menu.
+
+
+**<720px**\
+At his width you should use the most crucial functions in the secondary navigation. Avoid using a "Search "label after the icon.
+
+## Mobile Navigation
+Below the breakpoint of 1024px the navbar becomes mobile. This ensures that users with smaller devices like tablets or smartphones get the same user experience as the ones with larger devices.
+> By triggering the burger menu icon the menu icon becomes a cross icon. To apply the focus on usability and clarity all other elements except the logo and/or the application name become invisible.
+
+<img src={HeaderMobileNavigation} alt="Header Mobile Navigation" />
+
+## Sticky header
+In some situations our websites and applications provide a lot of content on a page and space on a screen is crucial. In those cases, we recommend to use a hidden header that only appears and stays sticky when the user scrolls/swipes up.
+
+### When to use
+- If the header contains important navigation or branding elements that the user needs to access quickly.
+
+### When not to use
+- If a website or application only has a few pages or a small amount of content.
+- If a page has a specific goal, such as completing a form or making a purchase, a sticky header can be distracting and take the user's attention away from the task. In this case, it may be better to deactivate the sticky behaviour or remove the entire header.<img src={ButtonGroupingByImportance} alt="Button Group By Importance" />
```

Comment:
ButtonGroupingByImportance was not imported (I added it from the Button Usage.mdx file imports)